lib/brakeman/warning.rb

```diff
@@ -175,7 +175,7 @@ def format_user_input strip = true
   def format_with_user_input strip = true, &block
     if self.user_input
       formatted = Brakeman::OutputProcessor.new.format(code, self.user_input, &block)
-      formatted.gsub!(/(\t|\r|\n)+/, " ") if strip
+      formatted = formatted.gsub(/(\t|\r|\n)+/, " ") if strip
```

Comment:
`formatted` is already going to be a fresh new string. Seems wasteful to create another new string right away :thinking: